pkg/transforms/kyverno.go

```diff
@@ -0,0 +1,39 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package transforms
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type KyvernoPolicyResource struct {
+	node Node
+}
+
+func KyvernoPolicyResourceBuilder(p *unstructured.Unstructured) *KyvernoPolicyResource {
```

Comment:
Please add link to the crd for quick reference. 
Is this the correct crd? https://github.com/kyverno/kyverno/blob/main/config/crds/kyverno/kyverno.io_policies.yaml

The crd has several printerColumns. Would it be useful to add those?

Reply:
Thanks! I added the CRD links and added a couple more columns. I don't think the others are needed at the moment.